data_steward/spec/pages/file_transfer_procedures.md

```diff
@@ -15,24 +15,40 @@ template: page
 1. Within five business days you should receive a response with:
      * Instructions on how to complete registration of your `pmi-ops.org` account
      * The name of your site's assigned bucket on Google Cloud Storage
-     * A test data set to be used during alpha
+     * A test data set to use in verifying your account.
      * Information on how to activate your account on our [JIRA site](https://precisionmedicineinitiative.atlassian.net) where you can report and track issues related to this process
   
       Keep this information in a safe place and do not share it with others.
 
-## Upload the Test Data Set
+## File Naming Convention
+
+`<Bucket name>_<quarter end date (yyyymmdd)>_<table name>.csv`
 
-__Note: You should not upload real participant data or de-identified data during alpha. You will be given a test data set to use instead.__
+An example is:
+* AoUSite123_20170930_person.csv
+* AoUSite123_20170930_visit_occurrence.csv
+* AoUSite123_20170930_condition_occurrence.csv
 
-The instructions below assume the bucket assigned to you was `test-site-bucket-aou-data-steward-nyc`. You should adapted according to the information that was sent to you by `sysadmin@pmi-ops.org`:
+## Prepare Files to be Uploaded
+
+For this test the same files and file structure will be used as in the [Data Sprint](https://cumc-dbmi.github.io/pmi_sprint_reporter/data_model.html).
```

Comment:
For new HPO sites, we should streamline experience from sprint to real EHR data submissions